Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -23,12 +23,13 @@ open Book_A_Desk.Infrastructure.DynamoDbEventStore
 type BookingsHttpHandler =
     {
         HandlePostWith: Models.Booking -> HttpHandler
-        HandleGetByEmailAndDate: unit -> HttpHandler
+        HandleGet: unit -> HttpHandler
     }
 
 module BookingsHttpHandler =
     let private notifyBooking output (booking : Models.Booking) notifySuccess errorHandler next context =
         task {
+            // remember me
```

Comment:
I don't think this comment is necessary, you can remove it

Reply:
All good. A good practice is to re-read your code when creating a PR, then you can spot these little comments that you have added during development and remove them before merging :)